spec/integration/outputs/index_spec.rb

```diff
@@ -69,6 +85,31 @@ def create_engine(engine_name, host, api_key)
         end
         expect(results.first.dig("message", "raw")).to eq "an event to index"
       end
+
```

Comment:
Can we add specs here to test against multiple events routing to different engines?